pom.xml

```diff
@@ -99,13 +100,49 @@
     <build>
         <pluginManagement>
             <plugins>
+                <plugin>
+                    <groupId>com.github.cafapi.plugins.docker.versions</groupId>
+                    <artifactId>docker-versions-maven-plugin</artifactId>
+                    <version>1.0.0-SNAPSHOT</version>
+                </plugin>
                 <plugin>
                     <groupId>io.fabric8</groupId>
                     <artifactId>docker-maven-plugin</artifactId>
                     <version>0.33.0</version>
                 </plugin>
             </plugins>
         </pluginManagement>
+        <plugins>
+            <plugin>
+                <groupId>com.github.cafapi.plugins.docker.versions</groupId>
+                <artifactId>docker-versions-maven-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <imageManagement>
+                        <image>
+                            <repository>${dockerHubPublic}/cafapi/opensuse-jre11</repository>
+                            <tag>4.0.0</tag>
+                            <digest>sha256:03c2b7fe78f3db1831a492ca341266eca7efb92f1f4968939308b9d038aba97c</digest>
+                        </image>
+                        <image>
+                            <repository>${dockerHubPublic}/cloudesire/webdav</repository>
+                            <tag>latest</tag>
+                        </image>
+                        <image>
+                            <repository>${dockerHubPublic}/library/postgres</repository>
+                            <tag>14.12-alpine3.19</tag>
```

Comment:
How will this ever be updated given the blanket ban on alpine images?

https://github.houston.softwaregrp.net/sepg/sepg-dependencies-command/blob/16133a89d7430d1508afae32823ad3d1d2629470/src/main/docker/command/ignoreDockerVersions.yaml#L28-L29

Reply:
Should we pick _bookworm_ or _bullseye_ instead? 
Remove "*alpine" from ignore versions?
Configure ignoreVersions per repository and not have "*alpine" ignored for postgres?

Reply:
Why are we wanting to ban alpine images?

Reply:
I added it for rabbitmq where were specifically looking for `management` tags and I wanted to include other tags in the ignore list.

Reply:
> I added it for rabbitmq...

Do we have a way in the ignore rules file to say that the rule applies just for a particular image or set of images?  If not then we'll need to raise a ticket and add that.

Reply:
No, will implement applying rules just for a particular image or set of images as part of [916303](https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=916303)